client_test.go

```diff
@@ -86,6 +88,9 @@ func TestStatsdClient(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// c.ExcludedTags = map[string]bool{"path": true}
```

Comment:
You mean the one I just removed in @a87910c ?
Just to be annoying to me, really, for being silly and forgetting leftovers.